skimage/restoration/_denoise.py

```diff
@@ -258,7 +258,7 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
 @utils.channel_as_last_axis()
 @utils.deprecate_multichannel_kwarg()
 @utils.deprecate_kwarg({'max_iter': 'max_num_iter'}, removed_version="1.0")
-def denoise_tv_bregman(image, weight=1.0, max_num_iter=100, eps=1e-3,
+def denoise_tv_bregman(image, weight=0.1, max_num_iter=100, eps=1e-3,
```

Comment:
Hi @lilsako, it may not have been clear, but the suggestion was to change the default to **5.0** which gives a result that is visually more similar to the `denoise_tv_chambolle` default. My intent was for both methods to give a visually similar output for the default value.

Reply:
Sorry for the misunderstanding, and thanks for the prompt response. I fixed it to 5.0 and waiting for the tests to pass!  Risako